Functions.Templates/Templates/CosmosDBTrigger-CSharp/build.config/template.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function"
+  ],
+  "name": "CosmosDBTrigger",
+  "identity": "Azure.Function.CSharp.CosmosDBTrigger",
+  "groupIdentity": "Azure.Function.CosmosDBTrigger",
+  "shortName": "CosmosDB",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "CosmosDBTrigger",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "Connection": {
+      "description": "The connection string to the service that contains the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "",
+      "replaces": "ConnectionValue"
+    },
+    "Database": {
+      "description": "This is the name of the database that includes the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "aDatabase",
+      "replaces": "DatabaseValue"
+    },
+    "Collection": {
+      "description": "This is the name of the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "aCollection",
+      "replaces": "CollectionValue"
+    },
+    "LeaseDatabase": {
+      "description": "This is the name of the database that includes the collection to store the leases.",
+      "type": "parameter",
+      "defaultValue": "aLeaseDatabase",
+      "replaces": "LeaseDatabaseValue"
+    },
+    "LeaseCollection": {
+      "description": "This is the name of the collection to store the leases.",
+      "type": "parameter",
+      "defaultValue": "aLeaseCollection",
+      "replaces": "LeaseCollectionValue"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "CosmosDBTriggerCSharp.cs"
+    }
+  ],
+  "defaultName": "CosmosDBTriggerCSharp"
```

Comment:
Add postAction object so that the required nuget dependencies are installed in VS when this template is added.

Make sure `Microsoft.NET.Sdk.Functions` added in the postAction Object. As per your template you would also need `Microsoft.Azure.WebJobs.Extensions.DocumentDB` package

Here is a sample 
https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Functions.Templates/Templates/ServiceBusQueueTrigger-CSharp/build.config/template.json#L57-L88